database_schema.sql

```diff
@@ -9,6 +9,19 @@ SET time_zone = "+00:00";
 -- --------------------------------------------------------
 
 --
+-- Table structure for table `blocks`
+--
+
+CREATE TABLE `blocks` (
```

Comment:
I would name this table a different name. I think blocks is too ambiguous and blocked_numbers would convey alot more